content/post/wrong-data.md

```diff
@@ -0,0 +1,55 @@
+---
+author: Randy Clinton
+title: "Wrong data can still be right"
+publishDate: "????"
+cover: "????"
+draft: true
+tags: 
+ - data-integrity
+ - communication
+ - errors
+---
+
+Let us explore what it means for data to be wrong. For now, think of it from the user’s reaction only. It is a data point, trend line, or report that makes the user lean into their screen, squint, tilt their head a bit, and say, “nooooo, that’s wrong.”  
```

Comment:
Use of the open imperative in the first sentence feels odd to me.

> Let us explore what it means for data to be wrong.

could become

> What does it mean when data are "wrong"?

or

> Analytics stakeholders often accuse data of being "wrong" even when they are not.

Reply:
Now that I think about it, probably better to re-work these lead-in paragraphs (what show up in the preview) to be a bit more thesis-driven (i.e., spelling out the conclusion of the post). Perhaps some of the language here would feel less odd if it were further down in the post.

Reply:
done, not pushed yet.